x-pack/plugins/security_solution/public/resolver/view/edge_line.tsx

```diff
@@ -65,16 +114,47 @@ const EdgeLineComponent = React.memo(
        */
       transform: `translateY(-50%) rotateZ(${angle(screenStart, screenEnd)}rad)`,
     };
-    return <div role="presentation" className={className} style={style} />;
+
+    let elapsedTimeLeftPosPct = 50;
+
+    /**
+     * Calculates a fractional offset from 0 -> 5% as magFactorX decreases from 1 to a min of .5
+     */
+    if (magFactorX < 1) {
+      const fractionalOffset = (1 / magFactorX) * ((1 - magFactorX) * 10);
+      elapsedTimeLeftPosPct += fractionalOffset;
+    }
+
+    return (
+      <StyledEdgeLine
+        role="presentation"
+        className={className}
+        style={style}
+        resolverEdgeColor={colorMap.resolverEdge}
+        magFactorX={magFactorX}
+      >
+        {elapsedTime && (
+          <StyledElapsedTime
+            backgroundColor={colorMap.resolverEdge}
+            leftPct={elapsedTimeLeftPosPct}
+            scaledTypeSize={scaledTypeSize}
+            textColor={colorMap.resolverEdgeText}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.resolver.elapsedTime"
+              defaultMessage="{duration} {durationType}"
```

Comment:
so in order to translate, you'd need to use a `select` here I guess? like:
```
"{durationType, select, day {días} weeks {semanas}}"
```

Reply:
It was my thought that FormattedMessage actually translates this internally from what I see here https://formatjs.io/docs/react-intl/components/#formattedmessage and in other parts of Kibana